source/common/modules/markdown-editor/context-menu/link-image-menu.ts

```diff
@@ -124,7 +124,15 @@ export function linkImageMenu (view: EditorView, node: SyntaxNode, coords: { x:
 
   showPopupMenu(coords, isLink ? linkTpl : imgTpl, (clickedID) => {
     if (clickedID === 'menu.copy_link') {
-      navigator.clipboard.writeText(url).catch(err => console.error(err))
+      var sanitizeUrl = url;
+      if(sanitizeUrl.includes("<") || sanitizeUrl.includes(">")){
```

Comment:
We can do this more elegantly. I would suggest to use a regexp (eg `/^<.+>$/`, but double check it does what you want) and then `url.slice` or `url.substring`. Right now the if looks a bit vague (it tries to replace < even if there's only a > in there, and since both signs belong semantically together, we should also treat them as together, ie a url that starts with < but does not end with > should not happen).

I hope I was clear in this statement ...? If not please ask! Otherwise, thanks for the suggestion, it looks otherwise good and mergeable!